frontend/components/layout/Header.vue

```diff
@@ -19,6 +19,8 @@
                     </label>
 
                     <div class="flex space-x-4">
+                        <button class="inline-block" @click="$emit('openRpcUrlInputModal')">Change RPC URL</button>
```

Comment:
Let's put it into the networks dropdown? Since this is also added to the regular "production version" of the website it should work well on mobile as well